packages/x-charts/src/PieChart/PieArcPlot.tsx

```diff
@@ -132,7 +130,6 @@ function PieArcPlot(props: PieArcPlotProps) {
               id={id}
               color={item.color}
               dataIndex={index}
-              highlightScope={highlightScope}
               isFaded={item.isFaded}
```

Comment:
Technically it's a breaking change. We can stop using them in our components, but slots should still get them

Reply:
But this property shouldn't even be used, nor should it have been passed down in the first place I guess. Since we already provide `isFaded/isHighlighted`. 🤔 

While I agree it is a breaking change, I just don't see how anyone would be using it.

Reply:
Ok yeah, now that I think of it, there would be other breaking changes in the current state as well. I've changed `useInteractionItemProps` signature.

Should I revert some changes but create deprecation warnings instead? 🤔 

```
// @deprecated
useInteractionItemProps(highlightScope: HighlightScope|undefined, skip?: boolean) useInteractionItemProps(skip?: boolean) 
```

Reply:
The modification of `useInteractionItemProps` is not an issue because it's not exported, so you can modify it as you want, user should not have access to this hook

https://github.com/mui/mui-x/blob/master/packages/x-charts/src/hooks/index.ts

Reply:
Oh, I didn't check because you mentioned it on the example of someone using it.